specs/2019-04/Windows-Plan.md

```diff
@@ -0,0 +1,98 @@
+
+> Note: this does not include handled exceptions and [custom properties](https://docs.microsoft.com/en-us/appcenter/diagnostics/features#key-value-pairs).  
+ 
+
+### Distribution 
+
+- Official support for WPF applications 
+ 
+> Note: this does not include in app updates through App Center.  
+### Analytics
+- Analytics SDK for WPF (.NET Framework) apps and all of its functionalities as listed in App Center's [analytics documentation](https://docs.microsoft.com/en-us/appcenter/analytics/).  
+
+## HockeyApp Comparison (Windows Specific) 
+### Diagnostics 
+Note: “full support” means supporting all App Center crashes features as indicated in App Center's [diagnostics documentation](https://docs.microsoft.com/en-us/appcenter/diagnostics/features), not including handled exceptions and custom properties. 
+
+
+|    | HockeyApp | App Center Today | Proposal | 
+| ---| --------- |:----------------:| ------- |
+| UWP| Yes       | Partial support | Full support for crashes |
+| WPF| Yes       | None | Full support for crashes |
+| WinRT / Silverlight | Yes       | None | None |
+| WinForms | (Open Source)    | None | Stretch goal |
+| Win32 | None | None | None |
+
+
+### Distribution  
+
+Note: HockeyApp supports over the air app updates for WPF, WinRT, and Silverlight applications. App Center will consider supporting these scenarios for distributing Windows apps using the .appInstaller file [#52](https://github.com/Microsoft/appcenter/issues/52)
+
+|    | HockeyApp | App Center Today | Proposal | 
+| ---| --------- |:----------------:| ------- |
+| UWP| Yes       | Yes | None (already supported) |
+| WPF| Yes       | None | Official support for .NET framework |
+| WinRT / Silverlight | Yes       | None | None |
+| WinForms | (Open Source)    | None | Stretch goal |
+| Win32 | None | None | None |
+
+### Analytics 
+
+Note: HockeyApp has user & event tracking in it’s analytics today. Full support in App Center means having a working App Center Analytics SDK with the full capabilities listed in App Center's [analytics documentation](https://docs.microsoft.com/en-us/appcenter/analytics/). 
+
+|    | HockeyApp | App Center Today | Proposal | 
+| ---| --------- |:----------------:| ------- |
+| UWP| Yes       | Yes | None (already supported) |
+| WPF| None       | None | Full support for .NET framework |
+| WinRT / Silverlight | Yes     | None | None |
+| WinForms | (Open Source)    | None | Stretch goal |
+| Win32 | None | None | None |
+
+
+## Scenarios for consideration: 
+
+Our team is considering support other features including, but not limited to the following: 
+
+- Handled exceptions for UWP apps [(#150)](https://github.com/Microsoft/appcenter/issues/150)
```

Comment:
Issue #150 is the number 2 most upvoted issue in this repo with 72 upvotes. What will it take for handled exceptions in UWP to be officially added in the spec?

Reply:
We are just starting to extend the support for Handled Errors to more platforms than just Xamarin so it is very likely that we'll do that. We first have to complete the "Full Support" of UWP as mentioned in the plan, then Handled Errors is likely next.

Reply:
Good to know! Thanks for the response.